lib/CPathfinder.h

```diff
@@ -21,6 +23,19 @@ struct TerrainTile;
 
 struct DLL_LINKAGE CGPathNode
 {
+	typedef EPathfindingLayer ELayer;
+
+	enum ENodeAction
```

Comment:
bool isGuarded?
Or BATTLE is not the only problematic state?

Nevertheless, my comment applies to entire struct, not just this enum.